support.html

```diff
@@ -9,15 +9,15 @@ <h1>Support Lab</h1>
         lower level classes with their assignments and topics. All the hours, and the current status, of the support
         lab can be found on this page.</p>
 
-    <h3>Winter 2019</h3>
-    <p>The support lab is now <strong>open</strong> for the Winter 2019 semester. Drop by OM1365 during one of the
-        scheduled sessions, or email us at <a href="mailto:supportlab@trucsclub.ca">supportlab@trucsclub.ca</a>.</p>
+    <h3>Winter 2020</h3>
+    
+    <p>The support lab is currently <strong> Closed on campus </strong> for the Winter 2020 semester due to the TRU campus shutdown. Find us on <a href="//trucsclub.ca/discord">Discord</a> though</p>
```

Comment:
What about "However, you can still get support online by joining our Discord chatroom! Find out how at https://trucsclub.ca/discord"

or something.

Reply:
applied suggestion